### PR TITLE
Agenda sync with Google Calendar

### PR DESCRIPTION
**Pull request by nasht00**

### CODE REVIEW

File: scripts/reversim-calendar/.gitignore [hunk 1]
```diff
@@ -0,0 +1,176 @@
+
+# Created by https://www.gitignore.io/api/node,jetbrains
```
Comment (rantav) [inline scripts/reversim-calendar/.gitignore]: That's a bit over-verbose IMO. 
We have a `.gitignore` file at the root of the project, can you please add just the entries you think are necessary to that file?

File: scripts/reversim-calendar/agenda.js [hunk 2]
```diff
@@ -0,0 +1,343 @@
+const axios = require("axios");
+const halls = ['A10', 'A3', 'A4', 'A5'];
```
Comment (rantav) [inline scripts/reversim-calendar/agenda.js]: I would add a comment that `halls`, `day1` and `day2` are copied from this and that files.

File: scripts/reversim-calendar/agenda.js [hunk 3]
```diff
@@ -0,0 +1,343 @@
...
+
+  loadSlot(slot, date) {
+    let events = [];
+    // Generate the startTime
+    let times = slot.time.split("-");
+    if (times.length < 2) {
+      //Not an actual event
+      return events;
+    }
+    let startTime = `${date}T${times[0]}:00+0300`;
```
Comment (rantav) [inline scripts/reversim-calendar/agenda.js]: I would put that `00+0300` in a const, e.g. 

```js
const tzStr = '00+0300';
```

File: scripts/reversim-calendar/agenda.js [hunk 4]
```diff
@@ -0,0 +1,343 @@
...
+        start: {
+          dateTime: startTime
+        },
+        end: {
+          dateTime: endTime
+        }
+      };
+
+      if(!slot.excludeHall){ // NOT Registration, Coffee, etc
+        // Single events are always in A10
```
Comment (rantav) [inline scripts/reversim-calendar/agenda.js]: > // Single events are always in halls[0]

File: scripts/reversim-calendar/agenda.js [hunk 5]
```diff
@@ -0,0 +1,343 @@
...
+            };
+
+            events.push(event);
+          }
+        }
+        count++;
+      }
+    }
+
+    //console.log(events);
```
Comment (rantav) [inline scripts/reversim-calendar/agenda.js]: delete this

**Comment by nasht00** [comment]
I have applied changes per the feedbacks

**Pull request merged**